imagetest/test_suites/network/mtu_test.go

```diff
@@ -0,0 +1,70 @@
+package network
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	gceMTU            = 1460
+	defaultInterface  = "eth0"
+	debian10Interface = "ens4"
```

Comment:
on my deb10 instance i have `enp0s4`, not `ens4`
maybe it varies depending on NIC type?

Reply:
Is this related to gvnic? i try instance with gvnic image for rhel and centos, it still has eth0. it looks like enp0s4 will be the nic name in the system with systemd

Reply:
no, the names are derived from the hardware. systemd can't change interface names. anyway, debian-10 has systemd so even so, debian10interface is not 'ens4'
suggest you run this test on all images to ensure you've got the right names, or else change the name logic